src/pylexibank/scaffold.py

```diff
@@ -6,9 +6,12 @@
 from pylexibank.metadata import LexibankMetadata
 
 
+TEMPLATES_DIR = pathlib.Path(pylexibank.__file__).parent / 'dataset_templates'
```

Comment:
This is not supposed to be exported, correct? (i.e., it is "private"). Should we rename it to `__TEMPLATES_DIR`?

Reply:
Following the logic above, i.e. having `__all__` declarations for anything public in modules, and aggregating these in `pylexibank.__init__`, it's clear that you probably shouldn't use `TEMPLATES_DIR`. But yes, we could rename it.